### ISSUE TITLE
Stencil: Nx Stencil generate application or library throwing "implementation is not a function" error.

### ISSUE THREAD

**Issue by vishnuskrish** [nxext/nx-extensions#1138]
**Describe the bug**
When creating a new stencil library or application using `$  nx generate @nxext/stencil:app my-app --verbose` command, it fails and throw the following error `implementation is not a function`.

![image](https://github.com/user-attachments/assets/c2881357-2463-465c-a62d-45099c3af697)

**To Reproduce**
* `$ npx create-nx-workspace@latest myorg`
* `$ cd myorg`
* `$ npm add @nxext/stencil --save`
* `nx generate @nxext/stencil:app my-app --verbose`
* code exited with the above error

**Expected behavior**
* should create the stencil app or library

**Additional context**

 NX   Report complete - copy this into the issue template

Node           : 20.17.0
OS             : win32-x64
Native Target  : x86_64-windows
npm            : 10.8.2

nx (global)    : 20.0.0
nx             : 20.0.0
@nx/js         : 20.0.0
@nx/jest       : 20.0.0
@nx/eslint     : 20.0.0
@nx/workspace  : 20.0.0
@nx/angular    : 20.0.0
@nx/cypress    : 20.0.0
@nx/devkit     : 20.0.0
@nx/react      : 20.0.0
@nx/storybook  : 20.0.0
@nx/web        : 20.0.0
@nx/webpack    : 20.0.0
typescript     : 5.5.4
---------------------------------------
Registered Plugins:
@nx/js/typescript

**Comment by pawel-twardziak** [comment]
Hi @vishnuskrish This has just got fixed there https://github.com/nxext/nx-extensions/pull/1139
Try again, the versions 20.0.1 has been released.
And remember to set the preset option (not `None` because `None` means `--preset=ts`) -> https://github.com/nxext/nx-extensions/issues/1137#issuecomment-2411975790

**Comment by pawel-twardziak** [comment]
@vishnuskrish btw next issue you are gonna be faced is that one https://github.com/nxext/nx-extensions/issues/1142 - eslint flat config is not yet supported by our libraries. I am currently working on this.

**Comment by pawel-twardziak** [comment]
@vishnuskrish could you re-check if the issue is still valid? It should get fixed.

**Comment by vishnuskrish** [comment]
ok. I will check again. Does the default preset (Typescript starter) works now?

**Comment by pawel-twardziak** [comment]
@vishnuskrish no, it is not supported yet either for official NX plugins or for @nxext plugins. You must use a preset other than `ts`

**Comment by vishnuskrish** [comment]
@pawel-twardziak I have checked with latest version. and the issue was not occurring.

**Comment by pawel-twardziak** [comment]
@vishnuskrish happy to hear that :))) can we close this issue then?

**Comment by pawel-twardziak** [comment]
@DominikPieper @edbzn this issue is ready to close.
